config/locales/en.yml

```diff
@@ -871,16 +864,16 @@ funding:
         <p>Next, this is just the first phase of a two year programme. <a href="https://en-gb.facebook.com/BigLotteryFund/">Follow us</a> or keep checking back to find out what’s next.</p>
         <p>Finally, if you’re from a small charity or small community group, we’re planning on offering you some <a href="/funding/programmes/digital-fund/assistance">free digital support from 2019</a></p>
       strand2: >
-        <p>Sorry, you’re ineligible</p>
-        <p>This kind of funding is only available to organisations that have already got a live service that’s being used to help a modest number of people, every week.</p>
+        <p>Sorry, you’re probably not eligible for Strand 2 funding.</p>
+        <p>This kind of funding is only available to organisations whose ideas are mature enough to be actually in operation, being used by people.</p>
```

Comment:
"to be actually in operation" might sound a bit nicer as "to actually be in operation", even if split infinitives are evil.